referenceframe/transformable.go

```diff
@@ -205,6 +205,9 @@ func (gF *GeometriesInFrame) Parent() string {
 
 // Geometries returns the geometries observed.
 func (gF *GeometriesInFrame) Geometries() map[string]spatialmath.Geometry {
+	if gF.geometries == nil {
+		return make(map[string]spatialmath.Geometry)
```

Comment:
Is there a reason we would not want to assign the newly allocated map to `gF.geometries` and let the function fall to the unconditional return statement?

Reply:
I tend to err against silently editing data structures that the user initialized

Reply:
This allows other, future functionality to also be able to distinguish between "this was initialized with nil" and "this was initialized with a zero-length map"